homeworks/roman.pavliuk_RomanPvlk/PokemonProject/index.html

```diff
@@ -0,0 +1,31 @@
+<!-- eslint-disable -->
```

Comment:
Hey, @RomanPvlk, please remove this disabling for eslint from this file. Anyway, your file is written in proper order and doesn't produce any errors.

Reply:
Got you, I'll do it today!